pombola/south_africa/views/organisations.py

```diff
@@ -63,6 +63,17 @@ def get_context_data(self, **kwargs):
         except EmptyPage:
             context['positions'] = paginator.page(paginator.num_pages)
 
+        # Determine if this org is a committee that can be contacted
+        queryset = models.Organisation.objects.filter(
+            pk=self.object.pk,
+            kind__name='National Assembly Committees',
+            contacts__kind__slug='email'
+        ).distinct()
+        if self.object in queryset:
+            context['contactable_via_writeinpublic'] = True
+        else:
+            context['contactable_via_writeinpublic'] = False
+
```

Comment:
Sorry to do that annoying thing of coming back to check fixups and seeing something else in the original PR, but ...

This is a surprising way to write this to me - is there a reason to do it this way rather than, say:
```
        context['contactable_via_writeinpublic'] = \
            self.object.kind.name == 'National Assembly Committees' \
            and self.object.contacts.filter(kind__slug='email').exists()
```

Reply:
Ahh that's much nicer! I tried something similar but it ended up more verbose because I didn't know about the `.exists()` method. Will make that change now 🙏

Reply:
@mhl I've fixed that in 688b82e480a6179a6c8065e8633b250b81d408c6